P0009/P0009.bs

```diff
@@ -2234,6 +2246,8 @@ template<class OtherElementType, class OtherExtents, class OtherLayoutPolicy, cl
     + `is_convertible_v<OtherLayoutPolicy::template mapping<OtherExtents>, mapping_type` is `true`,
     + `is_convertible_v<OtherAccessor, Accessor>` is `true`, and
     + `is_convertible_v<OtherAccessor::pointer, pointer>` is `true`.
+    + `OtherExtents::rank() == rank()` is `true`
```

Comment:
Couldn't we just say is_convertible_v<OtherExtents,Extents>? Or something like that?

Reply:
Yes, that seems better.